tests/src/test/scala/akka/kafka/internal/ConsumerSpec.scala

```diff
@@ -60,12 +60,17 @@ class ConsumerSpec(_system: ActorSystem)
 
   import ConsumerSpec._
 
-  def this() = this(ActorSystem())
+  def this() =
+    this(
+      ActorSystem("CommittingWithMockSpec",
+                  ConfigFactory
+                    .load()
+                    .withFallback(ConfigFactory.parseString("""akka.stream.materializer.debug.fuzzing-mode = on""")))
```

Comment:
Isn't this the wrong way around? The setting should overwrite the default in the Akka testkit.

Reply:
:+1: